platform/src/main/resources/anchor-config-defaults.yaml

```diff
@@ -44,10 +44,10 @@ stellar:
       type: logging-logback
       settings: logging-logback-settings          # The absolute location of the configuration data in this yaml file
 
-# ************************************
-# Application settings
-# ************************************
-app-config:
+#
+# Platform application configuration
+#
+platform-config:
```

Comment:
👀 💡  Hey, there's a lot of other places in the repo (including config files and documentation) that refer to `app-config`. We use `app-config` in stellar/kube as well.

If you're renaming it here, could you plz make sure you update all references to `app-config` throughout the project files and also stellar/kube? Otherwise, we will lose consistency around wording, which could make it more confusing to maintain this code.

Reply:
This is a leak of the coming up [PR#337](https://github.com/stellar/java-stellar-anchor-sdk/pull/337). 😞 

This is for integration tests and will not impact the stellar/kube. I had made a WIP PR to the stellar/kube too.

Reply:
Reverted back.